projects/documentation/content/_includes/partials/deprecation.njk

```diff
@@ -3,11 +3,14 @@
 >
 <div id="deprecation" class="spectrum-InLineAlert spectrum-InLineAlert--negative">
         <div class="spectrum-InLineAlert-header">
-          Deprecated component
+          Deprecation Warning
           <sp-icon-alert class="spectrum-InLineAlert-icon" dir="ltr" aria-hidden="true"></sp-icon-alert>
         </div>
         <div class="spectrum-InLineAlert-content">
-        This component has been deprecated.<br></br>
+          {% if isComponent %}
+              This component will be deprecated.
```

Comment:
Should this be `package` vs `component`?

Reply:
So you want to bifurcate as a package and a tool?

Reply:
Yes we can make something like `@spectrum-web-components/packages/banner` will be deprecated instead of `This component will be deprecated` maybe something like this?
<img width="1011" alt="Screenshot 2024-03-15 at 12 13 17 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/1fdebeda-84f5-4a23-a21b-50153f625361">
<img width="658" alt="Screenshot 2024-03-15 at 12 13 24 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/25c29624-a809-40df-ab1d-3f2d70a1ca79">